devices/develco.js

```diff
@@ -195,7 +196,10 @@ module.exports = [
         },
         exposes: [e.temperature(), e.battery(), e.smoke(), e.battery_low(), e.test(),
             exposes.numeric('max_duration', ea.ALL).withUnit('s').withValueMin(0).withValueMax(600).withDescription('Duration of Siren'),
-            exposes.binary('alarm', ea.SET, 'START', 'OFF').withDescription('Manual Start of Siren')],
+            exposes.binary('alarm', ea.SET, 'START', 'OFF').withDescription('Manual Start of Siren'),
```

Comment:
I think these exposes should be added to SMSZB-120 instead (since you added the fz.develco_genbinaryinput there)

Reply:
@Koenkk Ah, forgot - thanks for reminding me. Also updated the HESZB with fz.genbinaryinput and genbinaryinput to bind.